### PR TITLE
Install e2e script

### PR DESCRIPTION
**Pull request by gaius-qi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [x] Code compiles correctly.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by 244372610 — DISMISSED** [review]

File: Makefile [hunk 1]
```diff
@@ -297,10 +292,10 @@ help:
 	@echo "make build-dfget-man-page           generate dfget man page"
 	@echo "make test                           run unittests"
 	@echo "make test-coverage                  run tests with coverage"
-	@echo "make prepare-actions-e2e-test       prepare github actions E2E tests"
+	@echo "make install-actions-e2e-test       prepare github actions E2E tests environment"
```
Comment (jim3ma) [inline Makefile:295]: prepare -> install

File: hack/install-e2e-test.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      kind create cluster --config ${KIND_CONFIG_PATH}
+  fi
+}
+
+install-helm() {
+  if which helm >/dev/null ; then
+      helm install --wait --timeout 10m --create-namespace --namespace ${NAMESPACE} dragonfly ${CHARTS_PATH}
+  else
+      print_step_info "start install helm"
+      curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
```
Comment (jim3ma) [inline hack/install-e2e-test.sh:28]:
```
curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
chmod 700 get_helm.sh
./get_helm.sh
```

```
curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
```

File: hack/install-e2e-test.sh [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      chmod 700 get_helm.sh
+      ./get_helm.sh
+      helm install --wait --timeout 10m --create-namespace --namespace ${NAMESPACE} dragonfly ${CHARTS_PATH}
```
Comment (jim3ma) [inline hack/install-e2e-test.sh:31]: Move `helm install` out of `if`

File: hack/install-e2e-test.sh [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+curDir=$(cd "$(dirname "$0")" && pwd)
+cd "${curDir}/../" || return
+
+install-kind() {
+  if which kind >/dev/null ; then
+      kind create cluster --config ${KIND_CONFIG_PATH}
+  else 
+      print_step_info "start install kind"
+      go install sigs.k8s.io/kind@v0.11.1
+      kind create cluster --config ${KIND_CONFIG_PATH}
```
Comment (jim3ma) [inline hack/install-e2e-test.sh:19]: Move `kind create` out of `if`

File: hack/install-e2e-test.sh [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      helm install --wait --timeout 10m --create-namespace --namespace ${NAMESPACE} dragonfly ${CHARTS_PATH}
+  fi
+}
+
+install-ginkgo() {
+  if which ginkgo >/dev/null ; then
+      print_step_info "ginkgo has been installed"
+  else
+      print_step_info "start install ginkgo"
+      go get github.com/onsi/ginkgo/ginkgo
```
Comment (jim3ma) [inline hack/install-e2e-test.sh:40]: `go install`

File: hack/install-e2e-test.sh [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  fi
+
+  kind create cluster --config ${KIND_CONFIG_PATH}
+}
+
+install-helm() {
+  if which helm >/dev/null ; then
+  else
+      print_step_info "start install helm"
+      curl -fsSL get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | sh
```
Comment (jim3ma) [inline hack/install-e2e-test.sh:27]: curl -fsSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3

**Review by jim3ma — APPROVED** [review]

**Review by 244372610 — APPROVED** [review]

**Pull request merged**